provider/maas/constraints.go

```diff
@@ -29,7 +30,14 @@ func (env *maasEnviron) ConstraintsValidator(ctx context.ProviderCallContext) (c
 	if err != nil {
 		return nil, err
 	}
-	validator.RegisterVocabulary(constraints.Arch, supportedArches)
+
+	// Only consume supported juju architectures for this release. This will
+	// also remove any duplicate architectures.
+	lxdArches := set.NewStrings(supportedArches...)
```

Comment:
Looks like a cut & paste thing, using lxdArches as a variable name. Confused me at first when I went back to look at the code.

Reply:
I'll fix.